dub.sdl

```diff
@@ -0,0 +1,7 @@
+name "xlsxreader"
+description "A very simple xlsx reader"
+authors "Robert burner Schadek"
+copyright "Copyright © 2019, Robert burner Schadek"
+license "LGPL3"
+dependency "dxml" version="~>0.4.0"
+dependency "mir-core" version="~>1.1.109"
```

Comment:
please remove this unused dependency

Reply:
Removed at https://github.com/symmetryinvestments/xlsxreader/pull/16/commits/b43fc558c5b36f71a5657e351265f49f5a6a5ae8.